runtime/proto_errors.go

```diff
@@ -1,15 +1,24 @@
 package runtime
 
 import (
+	"context"
 	"io"
 	"net/http"
 
-	"context"
+        "github.com/grpc-ecosystem/grpc-gateway/internal"
```

Comment:
:eyes: spaces for indentation in a go file? :scream:

Reply:
Not sure what happened there. Ran `gofmt` on it.